### PR TITLE
Fix a few things in the new Keycloak Admin Client extension

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

File: bom/deployment/pom.xml [hunk 1]
```diff
@@ -198,6 +198,16 @@
...
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-keycloak-admin-client-deployment</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-keycloak-authorization-deployment</artifactId>
+                <version>${project.version}</version>
+            </dependency>
```
Comment (sberyozkin) [inline bom/deployment/pom.xml:210]: Thanks, my fault I missed it

File: extensions/keycloak-admin-client/runtime/src/main/resources/META-INF/quarkus-extension.yaml [hunk 2]
```diff
@@ -2,10 +2,9 @@
 name: "Keycloak Admin Client"
 metadata:
   keywords:
-  - "oauth2"
-  - "openid-connect"
```
Comment (sberyozkin) [inline extensions/keycloak-admin-client/runtime/src/main/resources/META-INF/quarkus-extension.yaml:6]: IMHO keeping `openid-connect` makes sense. We only this extension to help Keycloak users control Keycloak dynamically which is what they do with this admin client - but Keycloak is part of the OIDC setups. So if a KC user enters `openid-connect` this extension should be listed too

Comment (gsmet) [reply extensions/keycloak-admin-client/runtime/src/main/resources/META-INF/quarkus-extension.yaml:6]: @sberyozkin done

Comment (sberyozkin) [reply extensions/keycloak-admin-client/runtime/src/main/resources/META-INF/quarkus-extension.yaml:6]: @gsmet thanks

**Review by sberyozkin — APPROVED** [review]

**Pull request merged**